lib/Layout.js

```diff
@@ -132,6 +132,27 @@
 
 'use strict';
 
+const Buffer = require('buffer').Buffer;
```

Comment:
The [documentation](https://www.npmjs.com/package/buffer#usage) suggests this should be `... require('buffer/') ...` with a trailing slash.  Why isn't it?

Reply:
> To require this module explicitly, use require('buffer/') which tells the node.js module lookup algorithm (also used by browserify) to use the npm module named buffer instead of the node.js core module named buffer!

Without the trailing slash, the node module resolution algorithm will choose the core buffer module

Reply:
Ah; so that addresses my concern about mixing Buffer implementations?  If so, great; since the upstream module doesn't explain that clearly, please add a comment in the code, and note in the commit message that this change doesn't affect uses in the Node.js environment (other than adding an unnecessary external dependency, I guess...).

Reply:
Thanks for bringing up the extra external dependency, it's part of the reason I bumped the major version to v2. But I do agree it's not great. I will do some more exploring to figure out what the other viable alternatives would be.

Reply:
Removed the new dependency since bundlers are smart enough to polyfill the `require('buffer').Buffer` as is. I've separated the `buffer` module changes into a new commit